dll/win32/kernel32/winnls/string/locale.c

```diff
@@ -847,10 +1405,8 @@ LANGID WINAPI GetSystemDefaultUILanguage(void)
 LCID WINAPI LocaleNameToLCID( LPCWSTR name, DWORD flags )
 {
     struct locale_name locale_name;
-    static int once;
 
-    if (flags && !once++)
-        FIXME( "unsupported flags %x\n", flags );
+    if (flags) FIXME( "unsupported flags %x\n", flags );
```

Comment:
Seems we are losing actually some winesyncing code here, see commit https://github.com/reactos/reactos/commit/655d24d9eb530bcc12935bc8fa025025c78a23eb

Reply:
> Seems we are losing actually some winesyncing code here, see commit [655d24d](https://github.com/reactos/reactos/commit/655d24d9eb530bcc12935bc8fa025025c78a23eb)

I doubt that Wine3.3 will be the ultimate target of this PR. There will Sure be some follow ups. But despite that, you are right.

Reply:
I guess also that my underlying point is that with the extra commits we may loose track of what changed etc.

Reply:
We are temporarily losing some formatting and non-functional changes. If people would do proper winesyncs instread of randomly hacking around and turn wine shared files into a bigger mess of different versions each time, this wouldn't be a thing.